data/human/news.txt

```diff
@@ -1833,6 +2015,8 @@ news "spaceport video advertisement"
 			"A video of fast spaceships racing around a three dimensional course in space is shown on the screen, interwoven with clips of very luxurious ships where people appear to be watching the race."
 			"An action-packed advertisement about joining the Navy is playing on the screen. It shows impressive video-fragments of carriers, gunships, dropships and heroic marines."
 			"Some video is showing Syndicate equipment. The video ends with the slogan, 'Just what's needed to get the job done.'"
+			"Six days pirate hunting cruise, shoot real pirates for 2000 credits a day. 400 bullets included for free."
```

Comment:
```suggestion
			"Six days pirate on a hunting cruise, and shoot real pirates for 2000 credits a day. 400 bullets included, for free."
```

Reply:
I'll rewrite it a bit.

Reply:
Rewrote it to make it more clear that people go on a cruise to hunt pirates.

Reply:
That part looks good.